docusaurus/docs/learn/quickstarts/network/local-no-docker.md

```diff
@@ -58,16 +58,22 @@ ziti-controller started as process id: 1286. log located at: /home/vagrant/.ziti
 
 ### Verify the Controller is Running
 
-Assuming you have sourced the script, you will have an environment variable set named `$ZITI_EDGE_CONTROLLER_API`. After
-the controller has started, your controller should be listening at that hostname:port combination. You can see what your
-value is set to by running `echo $ZITI_EDGE_CTRL_ADVERTISED`. This variable defaults to: `$(hostname):1280`. Make sure the
-controller is on and listening and then start the edge router. 
+After running `expressInstall`, you will have environment variables set named `ZITI_CTRL_EDGE_ADVERTISED_ADDRESS` and 

```

Comment:
"After running `expressInstall`, you will have environment variables set"

Is that a change? Can we guarantee the script is able to export those variables? I'm not a bash ninja, but I didn't think we could do that?

Reply:
I'm not sure if it's true for all environments but it works in Mac, do you know if the env vars are set or not in Linux (WSL)?